modules/disable-quickstarts-admin-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="disable-quickstarts-admin-console_{context}"]
+= Disabling quick starts in the web console
+
+You can use the *Administrator* perspective of the web console to disable one or more quick starts.
+
+.Prerequisites
+
+* You have cluster administrator permissions and are logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administation* -> *Cluster Settings*.
+. In the *Cluster Settings* page, click the *Configuration* tab.
```

Comment:
```suggestion
. On the *Cluster Settings* page, click the *Configuration* tab.
```
Per ISG, use "on" for web pages: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#page